src/app/modules/timesheet/services/timesheet.service.ts

```diff
@@ -93,7 +93,7 @@ export class TimesheetService {
     return totalFlatFee;
   }
 
-  public getTimesheetsLocal() {
+  public getLocalstorageTimesheetsList() {
```

Comment:
`getLocalStorageTimesheetsList` because everywhere else you have an uppercase at `storage` word :)